store/postgres/src/writable.rs

```diff
@@ -1217,6 +1233,16 @@ impl Queue {
         Ok(map)
     }
 
+    fn get_range(
+        &self,
+        entity_types: Vec<EntityType>,
+        causality_region: CausalityRegion,
+        block_range: Range<BlockNumber>,
+    ) -> Result<BTreeMap<BlockNumber, Vec<EntityWithType>>, StoreError> {
+        self.store
+            .get_range(entity_types, causality_region, block_range)
+    }
```

Comment:
Maybe I lost track of what's happening, but didn't we talk about that this shouldn't go into the `WritableStore` and instead be part of a separate read interface? Sadly, I can't find my comment to that effect anymore; might be a victim of rebasing/force pushing

Reply:
Yeah, it was addressed in a different PR: https://github.com/graphprotocol/graph-node/pull/5544#discussion_r1870924333 . Unfortunately not merged/rebased downstream just yet.

Reply:
All should be up to date now.